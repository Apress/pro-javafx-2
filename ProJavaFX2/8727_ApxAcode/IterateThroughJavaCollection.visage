/*
 * @test
 * @run
 */

import java.util.*;

var list = new java.util.ArrayList();
list.add("One");
list.add("Two");
list.add("Three");

for (elem in list) {
  println("index: {indexof elem}, element: {elem}.");
}
