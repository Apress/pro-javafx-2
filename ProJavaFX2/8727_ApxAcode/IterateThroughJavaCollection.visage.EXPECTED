index: 0, element: One.
index: 1, element: Two.
index: 2, element: Three.
